recipe/meta.yaml

```diff
@@ -1,28 +1,22 @@
-{% set pip_name = 'avro' %}  # [py27]
-{% set pip_name = 'avro-python3' %}  # [py3k]
-{% set version2 = "1.9.2" %}  # {py27}
-{% set version3 = "1.9.2.1" %}  # {py3k}
+{% set pip_name = "avro-python3" %}
```

Comment:
```suggestion
{% set name = "avro-python3" %}
```

Reply:
@setu4993, since this isn't the name of the conda-forge package, I think it's better to keep this as it is.  This is also still the way things are on master:
https://github.com/conda-forge/python-avro-feedstock/blob/master/recipe/meta.yaml#L1

The same goes for your related suggestion below.

Reply:
Fair, thanks for considering.